feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/handling/NotificationHandlerExt.kt

```diff
@@ -73,28 +73,37 @@ fun Context.makePendingIntent(intent: Intent): PendingIntent {
         this,
         PEDDING_INTENT_REQUEST_CODE,
         intent,
-        PendingIntent.FLAG_UPDATE_CURRENT
+        PendingIntent.FLAG_UPDATE_CURRENT or PendingIntent.FLAG_IMMUTABLE
```

Comment:
Can you please remind me how FLAG_IMMUTABLE affects the behavior of pending intent for our case?

Reply:
This flag indicating that the created PendingIntent should be immutable. This means that the additional intent argument passed to the send methods to fill in unpopulated properties of this intent will be ignored.

I passed it since high API requires MUTABLE or IMMUTABLE flag.